nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees_desc_asb_[7,).partial.md

```diff
@@ -0,0 +1,15 @@
+In NServiceBus version 6 and above, the shape of the pipeline has changed so that a transport is in full charge of how a given transaction mode is implemented. A transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The Azure Service Bus transport only uses a transaction scope to implement the `SendAtomicWithReceive` transaction mode. Other modes work without any scope at all. The new architecture for `SendAtomicWithReceive` is schematically represented like this:
+
+![Transactions v7](transactions-v7.png)
```

Comment:
ps: I also didn't know until @danielmarbach came up with this solution, looks like many are unaware of how the core works. Maybe we should do some teaching to each other of how our parts work and why it is like that?